co-log-core/src/Colog/Core/Action.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE InstanceSigs #-}
```

Comment:
This extension is enabled by default for whole project. It's very useful! So I think that instead of adding it to every manually, you can just write:

```haskell
main = Doctest.doctest [ "-XInstanceSigs", "src" ]
```